lib/components/Modal.js

```diff
@@ -7,12 +7,15 @@ var elementClass = require('element-class');
 var renderSubtreeIntoContainer = require("react-dom").unstable_renderSubtreeIntoContainer;
 
 var SafeHTMLElement = ExecutionEnvironment.canUseDOM ? window.HTMLElement : {};
+var AppElement = document.body;
```

Comment:
@evoyy - yes, it would do nothing. That's the point on the server side, we're just creating a mock so that it doesn't try to access the DOM unless there's something there.